### ISSUE TITLE
[Feature request] (Chrome): Allow using existing extensions with new profile-directories

### ISSUE THREAD

**Issue by MathiasKoch** [common-fate/granted#62]
Hi!

First up thank you very much for an awesome tool!

It would be extremely nice if it would be possible to somehow keep extensions when opening the AWS console through granted.

I see the reason for it not being directly available is the internals of using a seperate `--profile-directory` for each profile/region permutation. Perhaps it could be possible to symlink the extensions of these new profiles to the Default profile `.config/google-chrome/Default` as an opt-in option in granted? Not sure if Chrome works that way, but i would guess you know more about that than me :)

**Comment by meyerjrr** [comment]
Hey @MathiasKoch thanks for the feature suggestion!

This definitely sounds achievable. We have tracked the issue and will investigate, aim to include it in the next release.

**Comment by azarudeena** [comment]
Yep I use https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers/ But not much of a fan of Firefox for various reasons. Would be really great if we can a extension to add.
